### PR TITLE
Checks for Makefile when validating a job

### LINKED ISSUE #12: validateJob does not check for a Makefile
Any TangoJob must have a Makefile since the autodriver runs `make` in order to start running a job. `validateJob` should ensure that a Makefile is part of the input files. If not, it should reject the job.

### PR DESCRIPTION
**Pull request by nayak16**
Confirm that the error message is appropriate. Not sure what the convention is. 
Resolves #12

### CODE REVIEW

**Comment by nayak16** [comment]
Seems like the previous commit got into this branch. My mistake just ignore that commit since it was already merged. Ill handle merge conflicts when it's ready to merge.

File: tangod.py [hunk 1]
```diff
@@ -227,6 +228,8 @@ def validateJob(job, vmms):
     """
     log = logging.getLogger('Server')
     errors = 0
+    print "JOB OBJECT RIGHT HERE ---------------------"
+    print str(job)
```
Comment (yrkumar) [inline tangod.py]: Remove this please.

**Comment by yrkumar** [comment]
Otherwise :+1: Remember to clean up the code before pushing.

**Comment by nayak16** [comment]
Merged e6edf70

**Pull request closed without merging**